docs/source/reference/loaders.rst

```diff
@@ -148,3 +148,29 @@ The data being loaded with the loader can be cropped/previewed prior to being
 passed along to the first method. The loader has the :code:`preview` parameter
 for configuring the cropping/previewing. Please see :ref:`previewing` for more
 details on previewing.
+
```

Comment:
I would actually move that to the top of the page. I sense that this will be used very often, almost as default setting. Could we may be get it [here](https://diamondlightsource.github.io/httomo/reference/loaders.html#basic-usage-of-the-standard-loader) somehow? cheers

Reply:
Good point, I've moved the `NXtomo` auto discovery section to the "basic usage" section.